app/src/main/java/com/github/se/polyfit/MainActivity.kt

```diff
@@ -64,6 +67,14 @@ class MainActivity : ComponentActivity() {
                 mealId = mealId)
           }
 
+          composable(Route.Settings) {
+            GenericScreen(
+                navController = navController,
```

Comment:
```
                navController = navController,
```
:(

Reply:
Yea, I know :( I followed the same format as the other files in this file. We can revamp later in its separate PR :D